File: flyrxbus/src/main/java/com/xia/flyrxbus/RxBusMessage.java
package com.xia.flyrxbus;

/**
 * @author xia
 * @date 2018/8/1.
 */
@SuppressWarnings({"WeakerAccess", "unused"})
public class RxBusMessage {
    public Object mObj;
    public Object[] mObjs;

    public RxBusMessage(Object... obj) {
        this.mObj = obj[0];
        this.mObjs = obj;
    }
}

File: flyrxbus/src/main/java/com/xia/flyrxbus/RxBusMessage.kt
package com.xia.flyrxbus

import androidx.annotation.NonNull

/**
 * @author xia
 * @date 2018/8/1.
 */
@Suppress("unused", "UNCHECKED_CAST")
class RxBusMessage(@NonNull vararg obj: Any) {
    @JvmField
    var mObj: Any = obj[0]
    @JvmField
    var mObjs: Array<out Any> = obj
}
